src/Common/utils.ts

```diff
@@ -1,8 +1,9 @@
 import { createHash } from 'crypto'
+
 import * as addressCodec from 'ripple-address-codec'
```

Comment:
The Xpring ESLint config has rules about import ordering and spacing.

Because `crypto` is a module built-in to Node, and `ripple-address-codec` is an external dependency, they are considered to be different "import groups" and thus require a blank line between them.